src/accounts/accounts.controller.ts

```diff
@@ -154,4 +156,42 @@ export class AccountsController {
       res.status(HttpStatus.NO_CONTENT).send({});
     }
   }
+
+  @ApiOperation({
+    summary: 'Freeze secondary accounts',
+    description: 'This endpoint freezes secondary accounts',
+  })
+  @ApiOkResponse({
+    description: 'Secondary accounts have been frozen',
+  })
+  @ApiUnauthorizedResponse({
+    description: 'Unauthorized',
+  })
```

Comment:
Add 
```
@ApiBadRequestResponse({
    description: 'The secondary Accounts are already frozen'
})
```

Reply:
I think it should be `422 Unprocessable entity`

Bad Request is more like sending a number where a string should be or are missing a field.

The SDK error should be `ErrorCode.UnmetPrerequisite` for this type of stuff